appeals/api/src/server/repositories/document.repository.js

```diff
@@ -27,6 +27,22 @@ export const getDocumentById = (guid) => {
 	});
 };
 
+/**
+ * @param {string[]} guids
+ * @returns {Promise<Document[]>}
+ * */
+export const getDocumentsByIds = (guids) =>
+	Promise.all(
+		guids.map(async (guid) => {
+			const doc = await getDocumentById(guid);
```

Comment:
Same here about performance. Probably this method could load them all by using something like
```
{ where: { guid in (guids) }}
```
Realistically there shouldn't be many attachments to a single comment, so again may be premature optimisation?

Reply:
I considered this but I think doing it that way will mean if one of the GUIDs isn't found, it will fail silently, which we might not want.

Reply:
good point!